test/osutils.jl

```diff
@@ -0,0 +1,8 @@
+@test (@static true ? 1 : 2) == 1
+@test (@static false ? 1 : 2) == 2
+@test (@static if true 1 end) == 1
+@test (@static if false 1 end) == nothing
+@test (@static true && 1) == 1
+@test (@static false && 1) == nothing
```

Comment:
This should actually return false? Similarly the `||` one below should return true.